<ui version="4.0" >
 <comment>*********************************************************************
**
** Copyright (C) 1992-2008 Trolltech ASA. All rights reserved.
**
** This file is part of the Qt Designer of the Qt Toolkit.
**
** This file may be used under the terms of the GNU General Public
** License versions 2.0 or 3.0 as published by the Free Software
** Foundation and appearing in the files LICENSE.GPL2 and LICENSE.GPL3
** included in the packaging of this file.  Alternatively you may (at
** your option) use any later version of the GNU General Public
** License if such license has been publicly approved by Trolltech ASA
** (or its successors, if any) and the KDE Free Qt Foundation. In
** addition, as a special exception, Trolltech gives you certain
** additional rights. These rights are described in the Trolltech GPL
** Exception version 1.2, which can be found at
** http://www.trolltech.com/products/qt/gplexception/ and in the file
** GPL_EXCEPTION.txt in this package.
**
** Please review the following information to ensure GNU General
** Public Licensing requirements will be met:
** http://trolltech.com/products/qt/licenses/licensing/opensource/. If
** you are unsure which license is appropriate for your use, please
** review the following information:
** http://trolltech.com/products/qt/licenses/licensing/licensingoverview
** or contact the sales department at sales@trolltech.com.
**
** In addition, as a special exception, Trolltech, as the sole
** copyright holder for Qt Designer, grants users of the Qt/Eclipse
** Integration plug-in the right for the Qt/Eclipse Integration to
** link to functionality provided by Qt Designer and its related
** libraries.
**
** This file is provided "AS IS" with NO WARRANTY OF ANY KIND,
** INCLUDING THE WARRANTIES OF DESIGN, MERCHANTABILITY AND FITNESS FOR
** A PARTICULAR PURPOSE. Trolltech reserves all rights not expressly
** granted herein.
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <class>qdesigner_internal::QtGradientEditor</class>
 <widget class="QWidget" name="qdesigner_internal::QtGradientEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>173</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="3" >
    <widget class="qdesigner_internal::QtGradientStopsEditor" name="gradientStopsEditor" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>13</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="linearPage" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_12" >
         <property name="text" >
          <string>Start X</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>Start Y</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_13" >
         <property name="text" >
          <string>Final X</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>Final Y</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1" >
        <widget class="QComboBox" name="spreadLinearComboBox" />
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Spread</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="endLinearYSpinBox" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="endLinearXSpinBox" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="startLinearYSpinBox" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="startLinearXSpinBox" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="radialPage" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_14" >
         <property name="text" >
          <string>Central Y</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_17" >
         <property name="text" >
          <string>Focal X</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_15" >
         <property name="text" >
          <string>Focal Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="centralRadialXSpinBox" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="centralRadialYSpinBox" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="focalRadialXSpinBox" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="focalRadialYSpinBox" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Radius</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="radiusRadialSpinBox" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="maximum" >
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label_20" >
         <property name="text" >
          <string>Spread</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QComboBox" name="spreadRadialComboBox" />
       </item>
       <item row="6" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_16" >
         <property name="text" >
          <string>Central X</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="conicalPage" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_18" >
         <property name="text" >
          <string>Central Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_19" >
         <property name="text" >
          <string>Central X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="centralConicalXSpinBox" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="centralConicalYSpinBox" >
         <property name="decimals" >
          <number>3</number>
         </property>
         <property name="maximum" >
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Angle</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="qdesigner_internal::QtDoubleSpinBox" name="angleConicalSpinBox" >
         <property name="wrapping" >
          <bool>true</bool>
         </property>
         <property name="decimals" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <double>360.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QComboBox" name="typeComboBox" />
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="qdesigner_internal::QtGradientWidget" name="gradientWidget" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qdesigner_internal::QtGradientStopsEditor</class>
   <extends>QFrame</extends>
   <header>qtgradientstopseditor.h</header>
  </customwidget>
  <customwidget>
   <class>qdesigner_internal::QtGradientWidget</class>
   <extends>QFrame</extends>
   <header>qtgradientwidget.h</header>
  </customwidget>
  <customwidget>
   <class>qdesigner_internal::QtDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qtspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>typeComboBox</tabstop>
  <tabstop>startLinearXSpinBox</tabstop>
  <tabstop>startLinearYSpinBox</tabstop>
  <tabstop>endLinearXSpinBox</tabstop>
  <tabstop>endLinearYSpinBox</tabstop>
  <tabstop>spreadLinearComboBox</tabstop>
  <tabstop>centralRadialXSpinBox</tabstop>
  <tabstop>centralRadialYSpinBox</tabstop>
  <tabstop>focalRadialXSpinBox</tabstop>
  <tabstop>focalRadialYSpinBox</tabstop>
  <tabstop>radiusRadialSpinBox</tabstop>
  <tabstop>spreadRadialComboBox</tabstop>
  <tabstop>centralConicalXSpinBox</tabstop>
  <tabstop>centralConicalYSpinBox</tabstop>
  <tabstop>angleConicalSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
